frontend/app/views/index.scala.html

```diff
@@ -211,7 +202,6 @@ <h3 class="h-intro">
             </div>
             <div class="listing__content">
                 <h4 class="h-intro h-intro--brand u-margin-bottom">Member levels</h4>
-                <p>Join as an annual Partner or Patron Member by <br/> 31 October 2015 and get 3 months free</p>
```

Comment:
Is there a reason why we are not mentioning the price changes on the homepage anymore?